public/index.html

```diff
@@ -28,10 +28,35 @@
 
     <link rel="apple-touch-icon" href="%PUBLIC_URL%/favicon-194x194.png" />
 
+    <link rel="stylesheet" type="text/css" href="css/preloader.css">
+
     <title>AsyncAPI Studio</title>
   </head>
   <body>
     <noscript>You need to enable JavaScript to run this app.</noscript>
+
+    <div class="preloader">
```

Comment:
Just out of interest really, but how come the .css file was added for this preloader? Can most of it be done using tailwind stuff we have?

Reply:
both yes and no. It's hard to do tailwind animations, especially rotating wheel, but I will try to remove redundant classes. Also I'm using this rotating wheel in several places, so easier for me is creating separate css class for that.

Reply:
Ok, removed :)